database/seed.js

```diff
@@ -23,21 +21,20 @@ let seed = async () => {
       let desc = faker.commerce.productDescription();
       let rating = Math.floor(Math.random() * Math.floor(5));
       let review_count = Math.floor(Math.random() * Math.floor(3000));
-      let price = faker.commerce.price(10, 200, 2, '$');
+      let price = Math.floor(Math.random() * Math.floor(1000));
       records.push([name,prodNum, photoUrl, desc, rating, review_count, price]);
     }
     csvWrite.writeRecords(records)
       .then( async () => {
-        await db.query(`COPY postgres("NAME", "PRODNUM", "PHOTOURL", "DESC", "RATING", "REVIEW_COUNT", "PRICE") FROM '/ndw001_SDC/database/csvFolder/data${loop}.csv' DELIMITER ',' CSV HEADER;`)
+        await db.query(`COPY products("name", "id", "photourl", "descript", "rating", "review_count", "price") FROM '/Users/nathan/Desktop/ndw001_SDC/database/csvFolder/data${loop}.csv' DELIMITER ',' CSV HEADER;`)
```

Comment:
Should use __dirname in the future to avoid putting your personal info on github